### PR TITLE
KLBP-433: Fixed user cannot proceed to next step due to a missing file.

### PR DESCRIPTION
**Pull request by iuliudascalu**

### CODE REVIEW

File: src/hooks/useAssociationStep1FormHook.ts [hunk 1]
```diff
@@ -55,10 +55,7 @@ export const useAssociationStep1FormHook = () => {
   const [isCopro, setIsCopro] = useState<string>(is_copro || '')
   const [isSiren, setIsSiren] = useState<string>(sirenValue as string)
 
-  if (
-    checkIfIsAutocertifOrEnfPassive(autocertif_recalcitrant, enf_passive) &&
-    initialState.length === 6
-  ) {
+  if (isAutocertifOrEnfPassive && initialState.length === 6) {
```
Comment (ioan-iacob-netea) [inline src/hooks/useAssociationStep1FormHook.ts:58]: Please create constants for all the values used (4, 5, 6, 7, 8).

**Review by ioan-iacob-netea — CHANGES_REQUESTED** [review]

File: src/hooks/useAssociationStep1FormHook.ts (hunk 1 above)
Comment (iuliudascalu) [reply src/hooks/useAssociationStep1FormHook.ts:58]: Can you plese suggest what name should have ?

Comment (mariuspop86) [reply src/hooks/useAssociationStep1FormHook.ts:58]: File_status?

Comment (iuliudascalu) [reply src/hooks/useAssociationStep1FormHook.ts]: Done.

**Review by ioan-iacob-netea — APPROVED** [review]

File: src/components/Journeys/Association/Step1/Step1DocumentsSection.tsx [hunk 2]
```diff
@@ -26,7 +26,7 @@ const Step1DocumentsSection: FC<IStep1AssociationHookData> = (data) => {
       <Step1UploadDocument {...data} />
       <StepNavButtons
```
Comment (mariuspop86) [inline src/components/Journeys/Association/Step1/Step1DocumentsSection.tsx:27]:
```suggestion
      <StepNavButtons
     {... {isNextDisabled, onClickNext, displayPrevious: false...}}
     
```

File: src/services/API/document.ts [hunk 3]
```diff
@@ -169,3 +169,11 @@ export async function deleteSecondaryPersonDocuments() {
     data: [{ typeId: '1', personTypeId: RL_PERSON_TYPE_ID }],
   })
 }
+
+export async function deleteDocument(typeId: string, personTypeId: string) {
```
Comment (mariuspop86) [inline src/services/API/document.ts:173]: Please update the function at line 165
```
export async function deleteSecondaryPersonDocuments() {
  return await deleteDocument('1', RL_PERSON_TYPE_ID)
} 
```

**Review by andreibuhus — APPROVED** [review]

File: src/components/Journeys/Association/Step1/Step1DocumentsSection.tsx (hunk 2 above)
Comment (iuliudascalu) [reply src/components/Journeys/Association/Step1/Step1DocumentsSection.tsx:27]: Done.

File: src/services/API/document.ts (hunk 3 above)
Comment (iuliudascalu) [reply src/services/API/document.ts:169]: Done.

File: src/services/API/document.ts [hunk 4]
```diff
@@ -163,9 +163,13 @@ export const parseGetDocumentResponse = (
 }
 
 export async function deleteSecondaryPersonDocuments() {
+  return deleteDocument('1', RL_PERSON_TYPE_ID)
```
Comment (mariuspop86) [inline src/services/API/document.ts:166]: '1' please use a constant

Comment (iuliudascalu) [reply src/services/API/document.ts]: Done.

**Review by mariuspop86 — APPROVED** [review]

**Pull request merged**